src/renderer/components/ft-channel-bubble/ft-channel-bubble.vue

```diff
@@ -42,6 +56,7 @@
     </div>
     <div
       :id="sanitizedId"
+      :hidden="!showTitle"
```

Comment:
I'm curious why you chose to use hidden here instead of a v-if, if it's because you want the element to be created but not shown, `v-show` would work too.

Reply:
Will change to `v-show`; forgot about that one. The element referenced by `aria-labelledby` needs the element to exist in the DOM.